tests/e2e/x/transfer/virtuous.go

```diff
@@ -126,8 +126,8 @@ var _ = e2e.DescribeXChainSerial("[Virtuous Transfer Tx AVAX]", func() {
 
 				needPermute := round > 3
 				if needPermute {
-					rand.Seed(time.Now().UnixNano())
-					rand.Shuffle(len(testKeys), func(i, j int) {
+					r := rand.New(rand.NewSource(time.Now().UnixNano())) //#nosec G404
+					r.Shuffle(len(testKeys), func(i, j int) {
```

Comment:
Why are we seeding the randomness here?
```suggestion
					rand.Shuffle(len(testKeys), func(i, j int) {
```

Reply:
fixed